paper.md

```diff
@@ -0,0 +1,95 @@
+---
+
+title: 'Hypothesis: A new approach to property-based testing'
+tags:
+    - Python
+    - testing
+    - test-case reduction
+    - test-case generation
+    - property-based testing
+
+authors:
+
+    - name: David R. MacIver
+      orcid: 0000-0002-8635-3223
+    - name: Zac Hatfield Dodds
+      orcid: 0000-0002-8646-8362
+    - name: many other contributors
+---
+
+# Summary
+
+*Property-based testing* is a style of testing popularised by the QuickCheck family of libraries,
+first in Haskell [@DBLP:conf/icfp/ClaessenH00] and later in Erlang [@DBLP:conf/erlang/ArtsHJW06],
+which integrates generated test cases into existing software testing workflows:
+Instead of tests which provide examples of a single concrete behaviour,
+tests specify properties that hold for a wide range of inputs,
+which the testing library then attempts to generate test cases to refute.
+For a general introduction to property-based testing, see [@PraiseOfPBT].
+
+Hypothesis is a mature and widely used^[It has over 100,000 downloads per week, over a thousand open source projects use it, and in 2018 more than 4% of Python users surveyed by the PSF reported using it.] property-based testing library for Python,
```

Comment:
These numbers are really impressive!  If you have any sources to back them up, then do cite them.

Reply:
Do we ever!

- https://pypistats.org/packages/hypothesis (download graphs)
- https://github.com/HypothesisWorks/hypothesis/network/dependents and https://libraries.io/pypi/hypothesis (dependents)
- https://www.jetbrains.com/research/python-developers-survey-2018/ (usage; crtl-f hypothesis)

It's pretty cool if I say so myself :grinning: